frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.unit.spec.js

```diff
@@ -160,7 +160,6 @@ describe("FieldValuesWidget", () => {
       it("should not call fetchFieldValues", () => {
         const { fetchFieldValues } = renderFieldValuesWidget({
           fields: [field],
-          fetchFieldValues,
```

Comment:
I think it was passed by mistake

Reply:
Was this breaking this test?

Reply:
yes, the value is used before it's defined with `const`